tools/library/CMakeLists.txt

```diff
@@ -35,6 +35,9 @@ include(GNUInstallDirs)
 set(CUTLASS_BUILD_MONO_LIBRARY OFF CACHE BOOL 
   "Determines whether the cutlass library is generated as a single file or multiple files.")
 
+option(CUTLASS_BUILD_SHARED_LIBS "Build shared libraries" ON)
+option(CUTLASS_BUILD_STATIC_LIBS "Build static libraries" OFF)
```

Comment:
For backward compatibility, the default for both of these will need to be `ON`, although we could decide to add a compatibility flag with message if we wanted that in the future we planned to make the default `OFF` for shared. I would save that extension for a separate request though.